src/sentry/static/sentry/app/views/organizationIncidents/detail/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+import LoadingError from 'app/components/loadingError';
+import {PageContent} from 'app/styles/organization';
+import withApi from 'app/utils/withApi';
+
+import IncidentHeader from './header';
+import IncidentDetailsBody from './body';
+
+class IncidentDetails extends React.Component {
```

Comment:
It would be nice to be consistent here, folder is `organizationIncidents/detail` vs `IndentDetails`

Reply:
What do you propose? Resource + Details is the pattern I saw most commonly in our codebase

Reply:
If that's the most common, then rename the directory `detail` -> `details`?